drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,77 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var entryPoint = client.get("http://example.com/").getEntryPoint();
```

Comment:
I don't think it's all that important. Despite the snippet is marked as JS, we should treat any examples here as pseudo-code.